ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -13,20 +13,56 @@ Test Teardown     Resources Test Teardown
 
 *** Variables ***
 @{LIST_OF_IDS_FOR_COMBINATIONS}         documentation--check-box    Red Hat managed--check-box
+${SUCCESS_STEP}              h3[normalize-space(@class="pf-c-title pf-m-md pfext-quick-start-task-header__title
+...                           pfext-quick-start-task-header__title-success")]
+
 
 
 *** Test Cases ***
-Verify Quick Starts Work As Expected
+Verify Quick Starts Work As Expected On Yes And Restart
+    [Documentation]   Verify the Quickstarts are completed successfully
+    ...    when all steps are marked as yes and restarted later
+    [Tags]  Sanity
+    ...     Tier1
+    ...     ODS-1306    ODS-1308
+    ...     ODS-1406    ODS-1405
+    Set Quick Starts Elements List Based On RHODS Type
+    Validate Number Of Quick Starts In Dashboard Is As Expected    ${quickStartElements}
+    Verify Quick Starts Work As Expected When All Steps Are Marked As Yes   ${quickStartElements}
+    Verify Quick Starts Work As Expected When Restarted And Left In Between    ${quickStartElements}
+    Verify Quick Starts Work As Expected When Restarting The Previous One    ${quickStartElements}
+
+
+Verify Quick Starts Work When All Steps Are Skipped
+    [Documentation]   Verify the Quickstarts work fine when all
+    ...    steps are skipped
+    [Tags]  Sanity
+    ...     Tier1
+    ...     ODS-1402
+    Set Quick Starts Elements List Based On RHODS Type
+    Validate Number Of Quick Starts In Dashboard Is As Expected    ${quickStartElements}
+    Verify Quick Starts Work As Expected When All Steps Are Skipped    ${quickStartElements}
+
+
+Verify Quick Starts Work When Atleast One Step Is Skipped
+    [Documentation]   Verify the Quickstarts work fine when atleast of the
+    ...    steps are skipped
+    [Tags]  Sanity
+    ...     Tier1
+    ...     ODS-1403
+    Set Quick Starts Elements List Based On RHODS Type
+    Validate Number Of Quick Starts In Dashboard Is As Expected    ${quickStartElements}
+    Verify Quick Starts Work As Expected When At Least One Step Is Skipped    ${quickStartElements}
+
+Verify Quick Starts Work As Expected When Atleast One Step Is Marked As No
+    [Documentation]   Verify the Quickstarts are works as expected
+    ...    when mark last one step as no
     [Tags]  Sanity
     ...     Tier1
-    ...     ODS-1166    ODS-1306
```

Comment:
what about ODS-1166? it's removed in this PR. Why?

Reply:
This got removed due to splitting of tests. Added back.